_includes/curation-workflow.md

```diff
@@ -0,0 +1,2164 @@
+
+---
+
+### Do the import on the command line
+
+Shell into the server you intend to import. See above for details on how to shell (ssh) into idr servers. Note that the commands below assume config edits.
+
+    ssh idr-testing-omero
+
+    ssh idr-next-omero
+
+Clone the study repository. To make sure that cases where the study contains additional metadata files required for the import, typically [patterns](https://github.com/IDR/idr0051-fulton-tailbudlightsheet/tree/master/patterns), [screens](https://github.com/IDR/idr0064-goglia-erkdynamics/tree/master/screens) or [companion files](https://github.com/IDR/idr0052-walther-condensinmap/tree/master/experimentA/companions), are taken care of, please always clone into the `/uod/idr/metadata`. This is a recomended "good practice" for all other cases as well:
+
+    cd /uod/idr/metadata/
+    sudo -Es git clone git@gitlab.com:idr/idr0092-ostrop-organoid.git
+    sudo -Es git pull origin master
+
+Sudo as the ``omero-server`` user. This is necessary for in-place import and for in-place upload of File Attachments.
+
+    sudo -u omero-server -s
+
+Still as ``omero-server`` user, start a
+[screen](https://kb.iu.edu/d/acuy)
+session so that if your connection to the server is lost stdout output
+from the import will not be lost. There are some useful hints about screen commands here below. The name of the screen should match the study name, as shown in example below:
+
+    screen -S idr0093
+
+Activate the virtual environment to be able to call ``omero`` commands from anywhere and to have a handy ``python`` environment to run scripts in.
+
+    source /opt/omero/server/venv3/bin/activate
+
+Define the OMERODIR (necessary for imports only).
+
+    export OMERODIR=/opt/omero/server/OMERO.server
+
+Log in to OMERO. The username for importing data on all 3 servers is ``demo``. Password is available in the OME credentials store (``pass``).
+
+    omero login demo@localhost
+
+To import Experiments:
+
+    omero import --bulk idr0048-abdeladim-chroms/experimentA/idr0048-experimentA-bulk.yml --file /tmp/idr0048.log  --errs /tmp/idr0048.err
+
+The whole output will go into the ``idr0048.err`` file (even when no errors are detected). Depending on your ``filePaths.tsv`` file, the import command will create the datasets with files inside them or a new Project with Datasets and Images in them. You might need to log in
+to the WebUI and move the datasets into the right project. See [Editing in the Web
+UI](#editing-in-the-web-ui) for more information.
```

Comment:
```suggestion

```

Reply:
The line break made the link to "Editing in web UI" not work. Hoping that removing the line break will fix that.